h2o-persist-gcs/src/main/java/water/persist/PersistGcs.java

```diff
@@ -225,4 +233,121 @@ public void close() throws IOException {
       }
     };
   }
+
+  @Override
+  public OutputStream create(String path, boolean overwrite) {
+    final GcsBlob gcsBlob = GcsBlob.of(path);
+    Log.debug("Creating: " + gcsBlob.getCanonical());
+    final WriteChannel writer = storage.create(gcsBlob.getBlobInfo()).writer();
+    return new OutputStream() {
+      @Override
+      public void write(int b) throws IOException {
+        ByteBuffer buffer = ByteBuffer.wrap(new byte[]{(byte) b});
```

Comment:
Are we sure there will always be byte ? 

Also, I was looking for other ways to write a single byte and not to wrap it with a new object each time a byte needs to be written - to no avail.

Reply:
It might be difficult because Google Cloud Storage library accepts only `ByteBuffer`.